.swiftlint.yml

```diff
@@ -17,36 +15,31 @@ disabled_rules:
   - syntactic_sugar
   - unused_capture_list
   - blanket_disable_command
+  - identifier_name
 
 opt_in_rules:
-  - empty_count
   - vertical_parameter_alignment_on_call
+  - opening_brace
```

Comment:
I think error severity is correct for this rule, instead of the default `warning` severity.

Reply:
There were a number of places where certain options were listed in two separate places resulting in a weird override.

Reply:
Yeah, but I think this change unintentionally lowered the severity of `opening_brace` from `error` to `warning`.